content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -240,17 +238,16 @@ TLS origination.
 1.  Change directory to the cloned repository:
 
     {{< text bash >}}
-    $ cd mtls-go-example
+    $ cd mtls-go-example || exit
     {{< /text >}}
 
 1.  Generate the certificates for `nginx.example.com`.
-    Use any password with the following command:
+    Run the following command, replacing `password` with your choice of password:
 
     {{< text bash >}}
-    $ ./generate.sh nginx.example.com <password>
+    $ yes | ./generate.sh nginx.example.com password
```

Comment:
Same here. Instead of putting the `yes | ` in the doc, you should put it in your test.sh file, in front of the call to the snip: `yes | snip_generate_client_and_server_certificates_and_keys_3`